Makefile

```diff
@@ -158,6 +158,7 @@ e2e-templates: $(addprefix $(E2E_TEMPLATES_DIR)/, \
 		 cluster-template-multi-az.yaml \
 		 cluster-template-multi-network.yaml \
 		 cluster-template-without-lb.yaml \
+		 cluster-template-flatcar.yaml \
```

Comment:
I would suggest to *not* add it here actually. This builds the cluster-template and puts them in `test/e2e/data/infrastructure-openstack` where the e2e code then finds them and injects the "ci-artifacts" scripts. Instead I would suggest to put this in a separate folder so that no special treatment is needed in the e2e code.